api/proto/auth/events.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/gravitational/trace"
+)
+
+func eventFromGRPC(in Event) (*services.Event, error) {
```

Comment:
Can you bring eventToGRPC in here as well but put in another package that both api and auth packages can import.

Reply:
I was able to move `eventToGrpc` to the same file, by moving both `events.go` and `client.go` to `/api`. I assume you only wanted it in a separate package to avoid an import cycle, right?